report-generator/src/main/java/com/teamscale/report/testwise/model/TestwiseCoverageReport.java

```diff
@@ -16,6 +16,11 @@ public class TestwiseCoverageReport {
 	/** The tests contained in the report. */
 	public final List<TestInfo> tests = new ArrayList<>();
 
+	@SuppressWarnings("unused") // Moshi might use this (TS-36140)
+	public TestwiseCoverageReport() {
```

Comment:
It is very likely that a defaut constructor is expected: see https://github.com/square/moshi/blob/master/moshi/src/main/java/com/squareup/moshi/ClassFactory.kt

Reply:
Having this default constructor is also a best practice recommended by the Moshi project: https://github.com/square/moshi/blob/master/README.md

Reply:
Why does `TestInfo` not need a default constructor then?

Reply:
Good catch. Also adjusted.